### PR TITLE
Apprenticeship assessment

### PR DESCRIPTION
**Pull request by trvllngmn**

### CODE REVIEW

File: data/discovery/register/assessment-organisation.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+fields:
+- assessment-organisation
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation.yaml]: assessment of what?

File: data/discovery/register/assessment-organisation.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- ukprn
+- organisation-name
+- assessment-organisation-type
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation.yaml]: again assessment of what?

File: data/discovery/register/assessment-organisation.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+fields:
+- assessment-organisation
+- ukprn
+- organisation-name
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation.yaml]: why not just name?

File: data/discovery/register/assessment-organisation.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+fields:
+- assessment-organisation
+- ukprn
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation.yaml]: whats a ukprn?

File: data/discovery/register/assessment-organisation.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+- organisation-name
+- assessment-organisation-type
+- address
+- start-date
+- end-date
+phase: discovery
+register: assessment-organisation
+registry: department-for-education
+text: 'A register of assessment organisations'
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation.yaml]: what is an assessment organisation and what is it assessing

File: data/discovery/register/assessment-organisation-type.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+fields:
+- assessment-organisation-type
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-type.yaml]: assessment of what

File: data/discovery/register/assessment-organisation-type.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+- organisation-type
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-type.yaml]: whats the difference between organisation type and assessment organisation type

File: data/discovery/register/assessment-organisation-type.yaml [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+- start-date
+- end-date
+phase: discovery
+register: assessment-organisation-type
+registry: department-for-education
+text: 'A register of type of assessment organisations'
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-type.yaml]: whats an assessment organisation and what is it assessing?

File: data/discovery/register/assessment-organisation-type.yaml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+fields:
+- assessment-organisation-type
+- organisation-type
+- start-date
+- end-date
+phase: discovery
+register: assessment-organisation-type
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-type.yaml]: assessment of what

File: data/discovery/register/assessment-organisation-type.yaml (hunk 8 above)
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-type.yaml]: whats an assessment organisation and what is it assessing?

File: data/discovery/register/assessment-organisation-standard-code.yaml [hunk 10]
```diff
@@ -0,0 +1,10 @@
+fields:
+- assessment-organisation-standard-code
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-standard-code.yaml]: assessment of what

File: data/discovery/register/assessment-organisation-standard-code.yaml [hunk 11]
```diff
@@ -0,0 +1,10 @@
...
+- standard-name
+- last-date-start
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-standard-code.yaml]: what is a last date start (there is no field description) and why not call it last-start-date it flows better?

File: data/discovery/register/assessment-organisation-standard-code.yaml [hunk 12]
```diff
@@ -0,0 +1,10 @@
...
+- start-date
+- end-date
+phase: discovery
+register: assessment-organisation-standard-code
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-standard-code.yaml]: assessment of what

File: data/discovery/register/assessment-organisation-standard-code.yaml [hunk 13]
```diff
@@ -0,0 +1,10 @@
...
+registry: department-for-education
+text: 'A register of standard codes of framework, of which apprenticehips are based upon'
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-standard-code.yaml]: The framework on which apprenticeships can be based? or have I mis-understood?

File: data/discovery/register/assessment-organisation-delivery-area.yaml [hunk 14]
```diff
@@ -0,0 +1,9 @@
+fields:
+- assessment-organisation-delivery-area
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-delivery-area.yaml]: assessment of what

File: data/discovery/register/assessment-organisation-delivery-area.yaml [hunk 15]
```diff
@@ -0,0 +1,9 @@
...
+- delivery-area
+- start-date
+- end-date
+phase: discovery
+register: assessment-organisation-delivery-area
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-delivery-area.yaml]: asessment of what

File: data/discovery/register/assessment-organisation-delivery-area.yaml [hunk 16]
```diff
@@ -0,0 +1,9 @@
...
+registry: department-for-education
+text: 'A register of delivery areas where assessment organisations deliver apprenticships'
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-delivery-area.yaml]: Delivery areas where assessment organizations deliver apprenticeships

File: data/discovery/register/assessment-organisation-apprenticeship.yaml [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+- assessment-organisation-apprenticeship
+- assessment-organisation
+- assessment-organisation-standard-code
+- assessment-organisation-delivery-area
+- start-date
+- end-date
+phase: discovery
+register: assessment-organisation-apprenticeship
+registry: department-for-education
+text: 'A register of apprenticeships offered per assessment organisation'
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-apprenticeship.yaml]: Apprenticeships offered per assessment organisation

Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-apprenticeship.yaml]: Apprenticeships offered per assessment organisation

File: data/discovery/register/assessment-organisation-apprenticeship.yaml [hunk 18]
```diff
@@ -0,0 +1,11 @@
+fields:
+- assessment-organisation-apprenticeship
+- assessment-organisation
+- assessment-organisation-standard-code
+- assessment-organisation-delivery-area
+- start-date
+- end-date
+phase: discovery
+register: assessment-organisation-apprenticeship
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-apprenticeship.yaml]: assessment of what

File: data/discovery/field/assessment-organisation-delivery-area.yaml [hunk 19]
```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: assessment-organisation-delivery-area
+phase: discovery
+register: assessment-organisation-delivery-area
+text: A reference to an area where an assessment organisation provides an apprenticeship.
```
Comment (Jamie-Atkinson) [inline data/discovery/field/assessment-organisation-delivery-area.yaml]: The area where an assessment organisation provides an apprenticeship.

File: data/discovery/field/last-date-start.yaml [hunk 20]
```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: datetime
+field: last-date-start
```
Comment (Jamie-Atkinson) [inline data/discovery/field/last-date-start.yaml]: last-start-date?

File: data/discovery/field/organisation-name.yaml [hunk 21]
```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: string
+field: organisation-name
+phase: discovery
+register: ''
+text: the name of an organisation.
```
Comment (Jamie-Atkinson) [inline data/discovery/field/organisation-name.yaml]: is this organisations? its a cardinality n field?

File: data/discovery/field/organisation-type.yaml [hunk 22]
```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
+datatype: string
+field: organisation-type
+phase: discovery
+register: ''
+text: a type of assessment organisation.
```
Comment (Jamie-Atkinson) [inline data/discovery/field/organisation-type.yaml]: A

File: data/discovery/field/organisation-name.yaml (hunk 21 above)
Comment (Jamie-Atkinson) [inline data/discovery/field/organisation-name.yaml]: The

File: data/discovery/register/assessment-organisation-standard-code.yaml [hunk 23]
```diff
@@ -0,0 +1,10 @@
+fields:
+- assessment-organisation-standard-code
+- standard-name
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-standard-code.yaml]: why not just use name field?

File: data/discovery/register/assessment-organisation-delivery-area.yaml [hunk 24]
```diff
@@ -0,0 +1,9 @@
+fields:
+- assessment-organisation-delivery-area
+- delivery-area
```
Comment (Jamie-Atkinson) [inline data/discovery/register/assessment-organisation-delivery-area.yaml]: isn't this just the name field?

File: data/discovery/field/standard-name.yaml [hunk 25]
```diff
@@ -0,0 +1,6 @@
+cardinality: 'n'
```
Comment (Jamie-Atkinson) [inline data/discovery/field/standard-name.yaml]: the data in this field in the data repo isnt cardinality n? it contains no ;

File: data/discovery/register/assessment-organisation.yaml (hunk 4 above)
Comment (trvllngmn) [reply data/discovery/register/assessment-organisation.yaml]: Added the following note to the README: 
`UKRPN` : according to the [UK Register of Learning Providers](https://www.ukrlp.co.uk/), a UKRPN is the 'UK Provide Reference Number'. This is the unique identifier used to share information with the UKRLP partner agencies.

**Pull request merged**